app/modules/assets/models.py

```diff
@@ -312,7 +373,8 @@ def is_mime_type_major(self, major):
 
     # Special access to the raw file only for internal users
     def user_raw_read(self, user):
-        return self.is_detection() and user.is_internal
```

Comment:
The `is_detection()` check was preventing Sage from downloading the asset when it needed it.  Opening this up is a security consideration, but is limited to internal users.